### ISSUE TITLE
Safari 11.0.3 shows blank screen

### ISSUE THREAD

**Issue by solikesimon** [netlify/netlify-cms#1780]
**Describe the bug**
Going to the demo (https://cms-demo.netlify.com) shows a blank page in Safari. Issue also occurs in private / incognito mode.

**To Reproduce**
Go to above link in Safari 11.0.3

**Expected behavior**
Admin page should appear

**Screenshots**
https://imgur.com/3dbARVj

**Applicable Versions:**

**CMS configuration**

**Additional context**

**Comment by tech4him1** [comment]
The exact version of the CMS demo site tested here is https://5baf807473f2cf0e68cf3861--cms-demo.netlify.com/.

**Comment by tech4him1** [comment]
Just grepping the Babel polyfills list for features added in Safari 10.1, seems like turning those polyfills on might be all that is necessary, assuming I'm reading it correctly:
```
ES6 Array.sort()
Object.entries()
Object.values()
Object.getOwnPropertyDescriptors()
```

I'll try to check into this and see if it's all we need.

**Comment by tech4him1** [comment]
I tried using the `useBuiltIns: 'usage'` option for Babel preset-env, but this seemed to break our build. I haven't tried it yet, but using `useBuiltIns: 'entry'` may work better.

If someone wanted to test this direction out before we get it integrated into the CMS, they could try adding the script `https://unpkg.com/core-js@2/client/core.js` to their index page, before the CMS script.

**Comment by AryanJ-NYC** [comment]
> If someone wanted to test this direction out before we get it integrated into the CMS, they could try adding the script https://unpkg.com/core-js@2/client/core.js to their index page, before the CMS script.

Hey @tech4him1. Tried this in Browserstack's Safari 10.1.2. No dice.
<img width="1227" alt="screen shot 2018-10-23 at 11 51 57 am" src="https://user-images.githubusercontent.com/8263430/47373597-2d798580-d6ba-11e8-8086-d58aca901b8a.png">

**Comment by tech4him1** [comment]
@AryanJ-NYC This issue was only for Safari 11.0.x, not Safari 10.x.x. We currently only support the latest two versions of Safari (at the time we release a new CMS version). If you'd like to request Safari v10 support, please open a new issue. Thanks!

**Comment by AryanJ-NYC** [comment]
@tech4him1 I tried the solution you proposed [above](https://github.com/netlify/netlify-cms/issues/1780#issuecomment-432298721) for Safari 11.0.x. That didn't work nor did messing around with the webpack and babel configurations.

**Comment by AryanJ-NYC** [comment]
@tech4him1 I tried the solution you proposed [above](https://github.com/netlify/netlify-cms/issues/1780#issuecomment-432298721] and that didn't work either. I also tried including [`@babel/polyfill`](https://babeljs.io/docs/en/babel-polyfill) using [`babel-preset-env`](https://babeljs.io/docs/en/babel-preset-env). That did not work either, unfortunately.

I've hit a wall and don't know where to go to try to fix this issue. Any thoughts? Thanks for your help!

**Comment by tech4him1** [comment]
@AryanJ-NYC What version of Safari were you testing on?

**Comment by AryanJ-NYC** [comment]
With respect to the comment before yours, 11.0.3.

**Comment by erquhart** [comment]
The latest release for Safari 11 is 11.1, that's the only version I have access to on Browserstack, and the demo works. If you upgrade to 11.1 and still have the issue please comment and we'll reopen.

**Comment by tech4him1** [comment]
@erquhart Even if not for this specific use-case, we certainly need to add polyfills to our build, since otherwise we are not always supporting all browsers. The only reason I haven't yet was problems getting it to work with our wacky build system, but if we are getting that fixed up now, we can look into polyfills once it is merged. If you want to create a tracking issue, that would be great.

AFAIK, we will either need to use the [`useBuiltIns`](https://babeljs.io/docs/en/babel-preset-env#usebuiltins) option on the babel `preset-env`, or a setting on `transform-runtime`. See this issue for more details on the problems I've had getting it set up: https://github.com/babel/babel/issues/6629.

**Comment by erquhart** [comment]
Understand, and agreed 👍 We can continue to use https://github.com/netlify/netlify-cms/issues/702 as the polyfills issue until this is resolved.
